src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $cart-border-color;
+  border-radius: 5px;
+  background-color: $cart-bg-color;
+  padding: $card-padding;
+
+  &__product-img {
+    display: flex;
+    justify-content: center;
+    padding: $product-img-padding 0;
+  }
```

Comment:
You have used `display: flex;` and `justify-content: center;` to center the image. It's a good practice for centering items horizontally in a container. However, since this is an image inside a box, you could use `margin: 0 auto;` to center the image. This would make your code cleaner and more understandable.